### PR TITLE
Update Pangolin to v1.1.4

### PR DESCRIPTION
**Pull request by DarianHole**
- Bump to version
- Added new param (-p --> Putative lineages)

### CODE REVIEW

File: tools/pangolin/pangolin.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
 
-pangolin '$input1' -t "\${GALAXY_SLOTS:-1}" --max-ambig '$additional.max_ambig' --min-length '$additional.min_length' $additional.write_tree &&
+pangolin '$input1' -t "\${GALAXY_SLOTS:-1}" --max-ambig '$additional.max_ambig' --min-length '$additional.min_length' $additional.write_tree $additional.include_putative &&
```
Comment (hellothisisMatt) [inline tools/pangolin/pangolin.xml:9]: Have you tested this tool when `include_putative` is true and false? Reason being is that some scripts may not like having a statement like this `myscript -a 1 -b 2 '' -c 5`, where there are `''` in the middle of the command.

File: tools/pangolin/pangolin.xml [hunk 2]
```diff
@@ -54,6 +61,7 @@ sed 's/,/\t/g' lineage_report.csv > lineage_report.tsv
             <param name="input1" value="test_seqs.fasta" />
             <section name="additional">
                 <param name="write_tree" value="true" />
+                <param name="include_putative" value="true" />
```
Comment (hellothisisMatt) [inline tools/pangolin/pangolin.xml:64]: Perhaps have a test where `include_putative` is false.

Comment (DarianHole) [reply tools/pangolin/pangolin.xml:64]: Default is set to false. In test one it should be tested with the blank statement as that test is all default settings correct?

Comment (hellothisisMatt) [reply tools/pangolin/pangolin.xml:64]: Ah yeah, didn't see test 1. Should default to false.

File: tools/pangolin/pangolin.xml (hunk 1 above)
Comment (DarianHole) [reply tools/pangolin/pangolin.xml:9]: Yep it works both ways! No `' '` in the input at all as those input parameters aren't surrounded with `' '`

File: tools/pangolin/pangolin.xml (hunk 2 above)
Comment (DarianHole) [reply tools/pangolin/pangolin.xml:64]: No worries, I doubled checked my planemo test and serve along with Travis CI tests and they behave/look as expected for command inputs

File: tools/pangolin/pangolin.xml (hunk 1 above)
Comment (hellothisisMatt) [reply tools/pangolin/pangolin.xml:9]: Oh interesting, was developing a wrapper and had `falsevalue=""` and it inserted a `''` in my command line statement.

**Pull request merged**